cpp/ps0001/main (2).cpp

```diff
@@ -2,9 +2,13 @@
 using namespace std;
 int main()
 {
-    short Value1,Value2,value3;
-    cout <<"Please enter the three values you wish to add together";
-    cin>>Value1>>Value2>>value3;
-    cout <<Value1+Value2+value3;
-return 0;
-}
\ No newline at end of file
+    // prompting user for 3 numbers to add 2/2
+    short value1, value2, value3;
+    cout << "Please enter the three values you wish to add together: \n";
+    cin >> value1 >> value2 >> value3;
+
+    // outputting the result of the summation process 3/3
+    cout << value1 + value2 + value3;
+    return 0;
+}
+// great work 😎 9/10 ⭐
```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code